stackit/internal/services/argus/scrapeconfig/resource.go

```diff
@@ -36,11 +36,19 @@ import (
 )
 
 const (
-	DefaultScheme                   = "https" // API default is "http"
-	DefaultScrapeInterval           = "5m"
-	DefaultScrapeTimeout            = "2m"
-	DefaultSampleLimit              = int64(5000)
-	DefaultSAML2EnableURLParameters = true
+	DefaultScheme                          = "https" // API default is "http"
+	DefaultScrapeInterval                  = "5m"
+	DefaultScrapeTimeout                   = "2m"
+	DefaultSampleLimit                     = int64(5000)
+	DefaultSAML2EnableURLParameters        = true
+	DefaultOauth2InsecureSkipVerify        = false
+	DefaultHttpSdConfigRefreshInterval     = "60s"
+	DefaultMetricsRelabelConfigAction      = "replace"
+	DefaultMetricsRelabelConfigRegex       = ".*"
+	DefaultMetricsRelabelConfigReplacement = "$1"
+	DefaultMetricsRelabelConfigSeparator   = ";"
+	DefaultHonorLabels                     = false
+	DefaultHonorTimeStamps                 = false
```

Comment:
I don't think there should be defaults for these new attributes
I was discussing with João and our guidelines for having hard-coded default values in the Provider are:
 - Field is required in the API and doesn't have a default
 - Field has a "bad" default (the DefaultScheme case in this file)
 - Exception for when a default value makes sense or was requested by the service team/user

 In general, the new fields here are Optional/Computed so they shouldn't have defaults